docs/installation_configuration_guide/DeploymentManagerConfiguration.md

```diff
@@ -14,24 +14,26 @@ deploymentConfig {
-| configExecutionOverrides   | config                                                  | {}                                 | see [below](#overriding-configuration-passed-to-runtime)                                                                                                                                                                                                            |
-| k8sDeploymentConfig        | config                                                  | {}                                 | see [below](#customizing-k8s-deployment)                                                                                                                                                                                                                            |
-| nussknackerInstanceName    | string                                                  | {?NUSSKNACKER_INSTANCE_NAME}       | see [below](#nussknacker-instance-name)                                                                                                                                                                                                                             |
-| logbackConfigPath          | string                                                  | {}                                 | see [below](#configuring-runtime-logging)                                                                                                                                                                                                                           |
-| commonConfigMapForLogback  | string                                                  | {}                                 | see [below](#configuring-runtime-logging)                                                                                                                                                                                                                           |
+`lite-k8s` Deployment Manager has the following configuration options:                 
+
+| Parameter                 | Type                                                | Default value                       | Description                                                                              |
+|---------------------------|-----------------------------------------------------|-------------------------------------|------------------------------------------------------------------------------------------|
+| mode                      | string                                              |                                     | Either streaming or request-response                                                     |
```

Comment:
I think there might be some more explanaintion on how there two modes works, but I'm not sure it's the best page for that

Reply:
I agree, but it should be somewhere in About like we discussed on some meeting before